frontend/src/components/mine/ContactInfo/ViewPartyRelationships.js

```diff
@@ -70,7 +70,6 @@ export class ViewPermittee extends Component {
 
   render() {
     const {
-      /* permittees, permitteeIds, mine */
     } = this.props;
 
```

Comment:
I believe you have to remove the entire
`const {
      /* permittees, permitteeIds, mine */
    } = this.props;`

Reply:
@nayr974